query-engine/connector-test-kit/src/test/scala/writes/topLevelMutations/DeleteMutationSpec.scala

```diff
@@ -70,8 +70,8 @@ class DeleteMutationSpec extends FlatSpec with Matchers with ApiSpecBase {
     server.queryThatMustFail(
       s"""mutation {deleteScalarModel(where: {unicorn: null}){unicorn}}""",
       project = project,
-      errorCode = 3040,
-      errorContains = "You provided an invalid argument for the where selector on ScalarModel."
+      errorCode = 0, // 3040,
+      errorContains = """"Error occurred during query execution:\nInterpretationError(\"Error for binding \\'0\\': RecordNotFound(\\\"Record to delete does not exist.\\\")\""""
```

Comment:
Seems like it should be user-facing with a code.

Reply:
👍